howso/utilities/installation_verification.py

```diff
@@ -336,8 +345,14 @@ def run_checks(self) -> int:  # noqa: C901
                     all_issues += 1
                     with open(log_file, mode="w+") as log:
                         print(f"Installation verification run: "
-                              f"{start_time.isoformat()}\n" + "=" * 80 + "\n",
+                              f"{start_time.isoformat()}\n",
                               file=log)
```

Comment:
`versions` is potentially unbound if `get_versions` fails. Need to initialize it outside the try/finally